x-pack/solutions/observability/plugins/observability/public/components/alert_actions/alert_actions.tsx

```diff
@@ -56,6 +62,20 @@ export function AlertActions({
   const userCasesPermissions = cases?.helpers.canUseCases([observabilityFeatureId]);
   const [viewInAppUrl, setViewInAppUrl] = useState<string>();
 
+  const alertAttachment = useMemo(() => {
+    return caseData?.comments?.find((comment) => {
+      if ('alertId' in comment && comment.alertId.includes(alert._id)) {
+        return comment.alertId.includes(alert._id);
+      }
+    });
+  }, [caseData, alert._id]);
+
+  const { showDeletionModal, onModalOpen, onConfirm, onCancel } = useDeletePropertyAction({
```

Comment:
IMO `useDeletePropertyAction` is more of a case-internal hook and should not need to be exported. I'll explain in another comment on the PR.